test/bidirectional_graph_test.rb

```diff
@@ -0,0 +1,166 @@
+require 'test_helper'
+
+require 'rgl/bidirectional_adjacency'
+require 'directed_graph_test'
+
+include RGL
+include RGL::Edge
+
+class TestBidirectionalAdjacencyGraph < TestDirectedGraph
```

Comment:
In https://github.com/monora/rgl/pull/76#discussion_r1105636080 I meant to define two separate test classes:

1. Your new class
2. The test class which inherits from TestDirectedGraph

I think, that this would increase readability. Perhaps providing a comment, that we simply reuse the tests from the super test class.

Reply:
I've been unable to make this work.

First of all, `TestDirectedGraph` has dependencies on `DirectedAdjacencyGraph` scattered throughout. It's not difficult to hoist them all up into `#setup`, but that would have to be done.

I tried it, but simply subclassing the `TestDirectedGraph` and overriding `#setup` did not work as expected. I'll push what I have but it's not right yet.

Reply:
For me that worked. See my changes in https://github.com/monora/rgl/tree/jsjuni-bidirectional-1